asus/zenbook/ux481/shared.nix

```diff
@@ -0,0 +1,23 @@
+{
+  config,
+  lib,
+  ...
+}:
+{
+  imports = [
+    ../../../common/cpu/intel/comet-lake/cpu-only.nix
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../battery.nix
+  ];
+
+  boot.kernelParams = [ # Enable suspend to work, otherwise the nvme will be mounted read-only on resume
+    "pcie_aspm=off"
+    "pcie_port_pm=off"
+    "nvme_core.default_ps_max_latency_us=0"
+    "mem_sleep_default=deep"
+  ];
```

Comment:
```suggestion
  boot.kernelParams = [ 
    # These options are needed for suspend to work, 
    # otherwise the nvme will be mounted read-only on resume
    "pcie_aspm=off"
    "pcie_port_pm=off"
    "nvme_core.default_ps_max_latency_us=0"
    "mem_sleep_default=deep"
  ];
```